CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2.4.0
```

Comment:
it might be too much to release a new version just for rubocop fixes. 
how about doing `Unreleased` for this PR, and making a new version later?

Reply:
what I meant is replacing `2.4.0` with `Unreleased` like [this](https://github.com/open-telemetry/opentelemetry-ruby/blob/f6efe3abccabbe1904058bce27d08c4a135649f7/sdk/CHANGELOG.md?plain=1#L3-L5), but I guess it's okay to add back in next PR 🙂

Reply:
ooooo !!
I am sorry, you did specifically mentioned `Unreleased` keyword, but I still misunderstood 🤦‍♀️ 
Thank you, made a note for the future, and fixed 👍